app/templates/settings/labs.hbs

```diff
@@ -53,6 +53,9 @@
                     {{#gh-feature-flag "subscribers"}}
                         Subscribers - Allow visitors to subscribe to e-mail updates of your new posts
                     {{/gh-feature-flag}}
+                    {{#gh-feature-flag "internalTags"}}
+                        Internal Tags - tags which don't show up in your theme.
```

Comment:
Can we update this, much like I did today for subscribers, to include a link to relevant documentation. It doesn't exist yet but will follow the same pattern as the other beta docs: `http://support.ghost.org/internal-tags-beta/`